include/vcpkg/base/json.h

```diff
@@ -200,12 +200,24 @@ namespace vcpkg::Json
 
     struct Object
     {
+        struct value_type
+        {
+            value_type(std::string sv, Value&& v) : key(std::move(sv)), value(std::move(v)) { }
+            value_type(std::string sv, const Value& v) : key(std::move(sv)), value(v) { }
```

Comment:
I think it would be better to avoid double string moves here. How about something like:

```suggestion
            template<class StringLike, class ValueLike>
            value_type(StringLike&& s, ValueLike&& v) : key(std::forward<StringLike>(sv)), value(std::forward<ValueLike>(v)) { }
```

Alternately, can this thing be an aggregate?